src/frontend/package.json

```
@@ -26,7 +26,9 @@
     "format-magnify:write": "cd magnify && yarn format:write",
     "format:check": "yarn format-magnify:check && yarn format-demo:check",
     "format:write": "yarn format-magnify:write && yarn format-demo:write",
-    "verify-all": "yarn format:check & yarn lint & yarn build & yarn test"
+    "verify-all": "yarn format:check & yarn lint & yarn build & yarn test",
```

Are you sure you want parallel processing here ?
Maybe you want to run commands only if the previous one succeed (personally I prefer this):
```suggestion
    "verify-all": "yarn format:check && yarn lint && yarn build && yarn test",
```
Or you want to run them all anyway
```suggestion
    "verify-all": "yarn format:check; yarn lint; yarn build; yarn test",
```

---

Since this is already merged, what do you think we should do @kernicPanel ? Commit this on this PR ? A new Issue just for this line (😅) ? Or maybe a `--fixup` on `main` ?

---

As this command is useful to check the generated components, I think we can fix it directly  in this PR.
Otherwise, a separate PR would be needed.

Doing a fixup on main will force us to make a choice:
- leaving this fixup commit in the repository history, which would be extremely ugly.
- squash the fixup on the related commit, so rewrite the main branch history, which could lead to problems with your coworkers.
So none of any choices are good, you should never do a fixup on a main branch.
They are only allowed on commit that can be squashed.
Let me know if something is not clear about this point. I think it's really important to get it.

---

That's perfectly clear and makes a lot of sense, thanks !